### DISCUSSION TITLE
405错误？

### DISCUSSION THREAD

**Discussion by lysanf** [messense/aliyundrive-webdav#134] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/25410552/140639460-754fecf1-d5d0-43b4-9f74-afb73a242248.png)
![image](https://user-images.githubusercontent.com/25410552/140639476-557f2a2e-5bf8-47f5-b2b3-2763617f0342.png)

**Comment by messense** [comment, marked as answer]
默认没有网页界面，请使用 WebDAV 客户端访问。简单的网页浏览测试界面通过 `-I` 参数开启。

https://github.com/messense/aliyundrive-webdav#%E5%91%BD%E4%BB%A4%E8%A1%8C%E7%94%A8%E6%B3%95

**Comment by fengchen-github** [comment]
啊，这个 -I 参数可以默认加上吧，哈哈
